census/templates/profile/_blocks/_stat_list.html

```diff
@@ -3,7 +3,7 @@
     <span class="{% if stat_class %}{{ stat_class }}{% else %}primary{% endif %}">
         <span class="value">
             {% if not stat.values.this %}
-                N/A
+                {% if WAZIMAP.na_label %}{{ WAZIMAP.na_label }}{% else %}N/A{% endif %}
```

Comment:
I added the default - much better, thanks.
(I wanted to ask you about the docs : )